### PR TITLE
Add fb composition layer sample project

### PR DESCRIPTION
**Pull request by devloglogan**
Using the same structure as #155, adds a sample project to showcase composition layers along with the functionalities of the related OpenXR extensions. Setting this as a draft for now as the passthrough sample should probably be merged first, in order to more easily ensure this PR follows the structural decisions we're making in the discussion of that PR.

### CODE REVIEW

File: samples/fb-composition-layers-sample/export_presets.cfg [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+name="Quest"
+platform="Android"
+runnable=true
+advanced_options=true
+dedicated_server=false
+custom_features=""
+export_filter="all_resources"
+include_filter=""
+exclude_filter=""
+export_path="../builds/FbPassthroughSample.apk"
```
Comment (m4gr3d) [inline samples/fb-composition-layers-sample/export_presets.cfg:12]: Looks like a leftover from the passthrough sample PR :)

Comment (devloglogan) [reply samples/fb-composition-layers-sample/export_presets.cfg]: Good catch, updated!

**Comment by devloglogan** [comment]
Passthrough sample has been merged so I've opened this PR!

**Review by dsnopek — APPROVED** [review]
I just tested this and it works great! The code looks good to me as well :-)

Great work!

**Pull request merged**